Automattic/jetpack
# Boost: Update Minify to use old Page Optimize file functions

Fixes #30791.

## Proposed changes:

* Updates Minify concat file functions to the old ones Page Optimize used before the changes introduced in https://github.com/Automattic/jetpack/pull/29352.

### Other information:


## Jetpack product discussion



p1684752302963189/1684746834.766979-slack-C016BBAFHHS

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






### Testing that it breaks properly
* Install the `trunk` version of the plugin;
* Enable CSS or JS concatenation;
* Your WP installation's File System should use `ftpext`. To force it to use it, add `define( 'FS_METHOD', 'ftpext' );` to `wp-config.php`;
* Check any page in the front end and it should be broken.


### Testing that it works properly
* Install this version of the plugin;
* Enable CSS or JS concatenation;
* Open a page and check that its source contains `_static` references and it works as expected.
